src/Microsoft.TestPlatform.Extensions.BlameDataCollector/IProcDumpArgsBuilder.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.Extensions.BlameDataCollector
+{
+    using System.Collections.Generic;
+
+    public interface IProcDumpArgsBuilder
```

Comment:
Made this a separate class/interface to make it easier to test

Reply:
And made sense to have this as a standalone interface rather than a private method in the blame utility class